vagrant-cfg.dist/common.yaml

```diff
@@ -192,3 +193,8 @@ hhvm::config::port: 9090  # make sure that hhvm and php5-fpm ports don't collide
 ## only fpm php.ini settings
 # php::fpm::settings:
 #   PHP/short_open_tag: 'On'
+
+## Wordpress config
```

Comment:
The SSL handling is a little bit complex. I think we should document at least here that the certs and keys are named after the vhost name? This is relevant if you want to set the paths to certs and keys in the source root.